subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -1457,6 +1480,10 @@ void radio_tmr_stop(void)
 	nrf_timer_task_trigger(SW_SWITCH_TIMER, NRF_TIMER_TASK_STOP);
 	nrf_timer_task_trigger(SW_SWITCH_TIMER, NRF_TIMER_TASK_SHUTDOWN);
 #endif /* !CONFIG_BT_CTLR_TIFS_HW */
+
+#if defined(CONFIG_SOC_SERIES_NRF54LX)
```

Comment:
Except this one (leave as is with the real SOC, as we don't have the POWER registers in the model yet for this target)